src/tri/_tri_wrapper.cpp

```diff
@@ -1,510 +1,58 @@
-    PyTriContourGeneratorType.tp_basicsize = sizeof(PyTriContourGenerator);
-    PyTriContourGeneratorType.tp_dealloc = (destructor)PyTriContourGenerator_dealloc;
-    PyTriContourGeneratorType.tp_flags = Py_TPFLAGS_DEFAULT;
-    PyTriContourGeneratorType.tp_methods = methods;
-    PyTriContourGeneratorType.tp_new = PyTriContourGenerator_new;
-    PyTriContourGeneratorType.tp_init = (initproc)PyTriContourGenerator_init;
-
-    return &PyTriContourGeneratorType;
-}
-
-
-/* TrapezoidMapTriFinder */
-
-typedef struct
-{
-    PyObject_HEAD
-    TrapezoidMapTriFinder* ptr;
-    PyTriangulation* py_triangulation;
-} PyTrapezoidMapTriFinder;
-
-static PyTypeObject PyTrapezoidMapTriFinderType;
-
-static PyObject* PyTrapezoidMapTriFinder_new(PyTypeObject* type, PyObject* args, PyObject* kwds)
-{
-    PyTrapezoidMapTriFinder* self;
-    self = (PyTrapezoidMapTriFinder*)type->tp_alloc(type, 0);
-    self->ptr = NULL;
-    self->py_triangulation = NULL;
-    return (PyObject*)self;
-}
-
-const char* PyTrapezoidMapTriFinder_init__doc__ =
-    "TrapezoidMapTriFinder(triangulation)\n"
-    "--\n\n"
-    "Create a new C++ TrapezoidMapTriFinder object.\n"
-    "This should not be called directly, instead use the python class\n"
-    "matplotlib.tri.TrapezoidMapTriFinder instead.\n";
-
-static int PyTrapezoidMapTriFinder_init(PyTrapezoidMapTriFinder* self, PyObject* args, PyObject* kwds)
-{
-    PyObject* triangulation_arg;
-    if (!PyArg_ParseTuple(args, "O!",
-                          &PyTriangulationType, &triangulation_arg)) {
-        return -1;
-    }
-
-    PyTriangulation* py_triangulation = (PyTriangulation*)triangulation_arg;
-    Py_INCREF(py_triangulation);
-    self->py_triangulation = py_triangulation;
-    Triangulation& triangulation = *(py_triangulation->ptr);
-
-    CALL_CPP_INIT("TrapezoidMapTriFinder",
-                  (self->ptr = new TrapezoidMapTriFinder(triangulation)));
-    return 0;
-}
-
-static void PyTrapezoidMapTriFinder_dealloc(PyTrapezoidMapTriFinder* self)
-{
-    delete self->ptr;
-    Py_XDECREF(self->py_triangulation);
-    Py_TYPE(self)->tp_free((PyObject *)self);
+PYBIND11_MODULE(_tri, m) {
+    py::class_<Triangulation>(m, "Triangulation")
+        .def(py::init<const Triangulation::CoordinateArray&,
+                      const Triangulation::CoordinateArray&,
+                      const Triangulation::TriangleArray&,
+                      const Triangulation::MaskArray&,
+                      const Triangulation::EdgeArray&,
+                      const Triangulation::NeighborArray&,
+                      bool>(),
+            py::arg("x"),
+            py::arg("y"),
+            py::arg("triangles"),
+            py::arg("mask"),
+            py::arg("edges"),
+            py::arg("neighbors"),
+            py::arg("correct_triangle_orientations"),
+            "Create a new C++ Triangulation object.\n"
+            "This should not be called directly, instead use the python class\n"
```

Comment:
```suggestion
            "This should not be called directly, use the python class\n"
```
(or drop the instead on the next line.)